test/unit/models/embassy_test.rb

```diff
@@ -42,7 +42,6 @@ class EmbassyTest < ActiveSupport::TestCase
     location = Embassy.new(toytown)
 
     assert_equal "Legoland", location.remote_services_country.name
-    assert_equal "British Embassy Legoland", location.offices.first.title
```

Comment:
Minor: Can you explain in the commit note *why* it's OK to remove this assertion?

Reply:
I've now added additional explanation in the commit message and force-pushed.